2_process.R

```diff
@@ -0,0 +1,19 @@
+source("2_process/src/filter_wqp_data.R")
+source("2_process/src/munge_nwis_SpC_data.R")
+
+p2_targets_list <- list(
+  # Filter harmonized WQP data for salinity data
+  tar_target(
+    p2_filtered_wqp_data,
+    filter_wqp_salinity_data(p1_wqp_data,major_ion_names,select_wqp_vars,omit_wqp_events)
+  ),
+  # Subset and save discrete SpC data from harmonized WQP
+  tar_target(
+    p2_wqp_spC_csv,
+    subset_wqp_spC_data(p2_filtered_wqp_data,fileout="2_process/out/DRB_WQdata_SpC_data.csv")),
```

Comment:
Could remove NA values. They are all listed as equipment maintenance or *** provisional codes

Reply:
Sorry, placed this comment on the wrong dataset. Supposed to be the DRB_daily_SpC_data.csv

Reply:
Good point! This relates to a question I had for the continuous data too - will we want padded time series, or just the available data? Does it make a difference from a modeling perspective?

Reply:
Questions about time series formatting migrated to discussion (#15). I've gone ahead and removed NA values from the daily data summaries since they all reference maintenance or other equipment codes.